### ISSUE TITLE
[Feature]Is there any plan to support Dashboard customization

### ISSUE THREAD

**Issue by hxbb00** [jobrunr/jobrunr#18]
I need to customize the Dashboard ,do you has plan to support ?

**Comment by rdehuyss** [comment]
Could you explain what you want to customize? I need more context to understand what you want to achieve.

**Comment by hxbb00** [comment]
For example, internationalization, or remove part of the button,like github

**Comment by rdehuyss** [comment]
I like the idea of internationalization but that won't be for release 2.0 or something.

Regarding the removal of the GitHub button, I'm a little bit skeptical as it is not in accordance with the License. If you want, we can discuss further on Gitter (https://gitter.im/jobrunr/community).

**Comment by hxbb00** [comment]
Cloud you consider providing a mechanism for override the dashboard  `JobRunrStaticFileHandler`?
like :
```java
public JobRunrDashboardWebServer(StorageProvider storageProvider, JsonMapper jsonMapper, int port) {
        RedirectHttpHandler redirectHttpHandler = new RedirectHttpHandler("/", "/dashboard");

  // override the StaticFileHandler
        JobRunrStaticFileHandler staticFileHandler = createStaticFileHandler();

        JobRunrApiHandler dashboardHandler = new JobRunrApiHandler(storageProvider, jsonMapper);
        JobRunrSseHandler sseHandler = new JobRunrSseHandler(storageProvider, jsonMapper);

        teenyWebServer = new TeenyWebServer(port);
        teenyWebServer.createContext(redirectHttpHandler);
        teenyWebServer.createContext(staticFileHandler);
        teenyWebServer.createContext(dashboardHandler);
        teenyWebServer.createContext(sseHandler);
        teenyWebServer.start();

        LOGGER.info(format("JobRunr dashboard started at http://%s:%d%s",
                teenyWebServer.getWebServerHostAddress(),
                teenyWebServer.getWebServerHostPort(),
                staticFileHandler.getContextPath()));
    }
   
     protected JobRunrStaticFileHandler createStaticFileHandler(){
         return new JobRunrStaticFileHandler();
     }

```

**Comment by rdehuyss** [comment]
I made it possible to override the different file handlers but made them package private.

This as I can not guarantee 100% that this will be backwards compatible. But, I'll do my best ;-)

**Comment by hxbb00** [comment]
thank you!

**Comment by rdehuyss** [comment]
Solved in commit 22b1e9e6ff0c07c6dc05e9d09922e1810e3b9d5b.
